src/main/java/step2/controller/LottoController.java

```diff
@@ -5,22 +5,29 @@
 import step2.view.InputView;
 import step2.view.RenderView;
 
+import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class LottoController {
     public static void run() {
         LottoDto lottoDto = InputView.inputPurchaseMoney();
 
         Money myMoney = Money.of(lottoDto.getPurchaseMoney());
 
-        List<Lotto> lottoList = LottoIssuer.issueLottos(myMoney);
-        PurChasedLotto purChasedLotto = new PurChasedLotto(lottoList);
+        LottoDto lottoDto2 = InputView.inputCustomLottoNumberList();
 
-        RenderView.showLottoList(lottoList);
+        List<Lotto> nonAutoLottoList = LottoIssuer.issueNonAutoLotto(lottoDto2.getNonAutoLottoNumberList());
+        List<Lotto> autoLottoList = LottoIssuer.issueAutoLottos(myMoney.getNumberOfPurchases() - nonAutoLottoList.size());
 
-        LottoDto lottoDto2 = InputView.inputWinningLottoNumber();
+        PurChasedLotto purChasedLotto = new PurChasedLotto(nonAutoLottoList, autoLottoList);
```

Comment:
구매하다 라는 purchase 단어는 pur + chase가 합쳐진 단어인가요?
하나의 단어로 알고있기에 카멜케이스를 적용하지 않아도 될 것 같아요 :)
```suggestion
        PurchasedLotto purchasedLotto = new PurchasedLotto(nonAutoLottoList, autoLottoList);
```